core-services/prow/02_config/openshift-assisted/assisted-installer-ui/_prowconfig.yaml

```diff
@@ -4,10 +4,12 @@ branch-protection:
       repos:
         assisted-installer-ui:
           branches:
-            ^release-.*:
-              protect: false
             ^releases/v.*:
```

Comment:
this won't work either, right?

Reply:
I think it won't, yeah. I left it there though, as currently nothing is being forwarded to those `^releases/v.*` branches anyway, and I'm currently talking with the owners and I'm not sure if they need branch protection at all, so maybe it will just get removed.

Reply:
slack thread: https://redhat-internal.slack.com/archives/CBN38N3MW/p1742286859084419